notification-portlet-webcomponents/notification-banner/src/components/NotificationBanner.vue

```diff
@@ -1,58 +1,76 @@
 <template>
-  <div class="hello">
-    <h1>{{ msg }}</h1>
-    <p>
-      For guide and recipes on how to configure / customize this project,<br>
-      check out the
-      <a href="https://cli.vuejs.org" target="_blank" rel="noopener">vue-cli documentation</a>.
-    </p>
-    <h3>Installed CLI Plugins</h3>
-    <ul>
-      <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel" target="_blank" rel="noopener">babel</a></li>
-      <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
-    </ul>
-    <h3>Essential Links</h3>
-    <ul>
-      <li><a href="https://vuejs.org" target="_blank" rel="noopener">Core Docs</a></li>
-      <li><a href="https://forum.vuejs.org" target="_blank" rel="noopener">Forum</a></li>
-      <li><a href="https://chat.vuejs.org" target="_blank" rel="noopener">Community Chat</a></li>
-      <li><a href="https://twitter.com/vuejs" target="_blank" rel="noopener">Twitter</a></li>
-      <li><a href="https://news.vuejs.org" target="_blank" rel="noopener">News</a></li>
-    </ul>
-    <h3>Ecosystem</h3>
-    <ul>
-      <li><a href="https://router.vuejs.org" target="_blank" rel="noopener">vue-router</a></li>
-      <li><a href="https://vuex.vuejs.org" target="_blank" rel="noopener">vuex</a></li>
-      <li><a href="https://github.com/vuejs/vue-devtools#vue-devtools" target="_blank" rel="noopener">vue-devtools</a></li>
-      <li><a href="https://vue-loader.vuejs.org" target="_blank" rel="noopener">vue-loader</a></li>
-      <li><a href="https://github.com/vuejs/awesome-vue" target="_blank" rel="noopener">awesome-vue</a></li>
-    </ul>
+  <div class="notification-banner">
+    <b-alert varient="info" show>
```

Comment:
@ChristianMurphy -- The `<b-alert>` component appears unstyled when I run it with `npm run serve` (not like the examples).  I'm thinking it's because...
  - (1) Bootstrap is not applied to the `index.html` page (the basis of `run serve`);  and
  - (2) `bootstrap-vue` does not bake-in Bootstrap (they expect you to add it)

Reply:
Yep, from the docs https://bootstrap-vue.js.org/docs#webpack

```
import 'bootstrap/dist/css/bootstrap.css'
import 'bootstrap-vue/dist/bootstrap-vue.css'
```

Reply:
Where does that go?  (I don't see that anywhere in `notification-modal`.)

Reply:
In the JavaScript, Webpack takes are of the rest.

> I don't see that anywhere in notification-modal

Good observation, notification modal takes a slightly different approach, it custom builds bootstrap with just the styles needed to render the page. (for smaller bundle size)

https://github.com/Jasig/NotificationPortlet/blob/7356d132eda50699d588200de185c2e5bb88ddc6/notification-portlet-webcomponents/notification-modal/src/components/NotificationModal.vue#L165-L173

This component will get to that point, but for early development, importing everything is fine.